README.md

```diff
@@ -26,116 +22,49 @@ Package Info
 - versioning: [http://semver-ftw.org](http://semver-ftw.org)
 
 
-Usage
------
+## Usage
 
 ### Installation
 
     npm install --save bip38
 
 
-API
----
-
-### Bip38([versions])
-
-Constructor that creates a new `Bip38` instance. 
-
-- **versions**: optional parameter to set the versions. Defaults to Bitcoin.
-
-
-### versions
-
-A field that accepts an object for the address version. This easily allows you to support altcoins. Defaults to Bitcoin values.
-
-
-**example:**
-
-```js
-var Bip38 = require('bip38')
-
-var privateKeyWif = '5KN7MzqK5wt2TP1fQCYyHBtDrXdJuXbUzm4A9rKAteGu3Qi5CVR'
-
-var bip38 = new Bip38()
-
-// not necessary, as Bitcoin is supported by default
-bip38.versions = {
-	private: 0x80, 
-  public: 0x0
-}
-bip38.encrypt(privateKeyWif, "super-secret", "1Jq6MksXQVWzrznvZzxkV6oY57oWXD9TXB"})
-```
-
-### scryptParams
-
-A field that accepts an object with the follow properties: `N`, `r`, and `p` to control the [scrypt](https://github.com/cryptocoinjs/scryptsy). The
-BIP38 standard suggests `N = 16384`, `r = 8`, and `p = 8`. However, this may yield unacceptable performance on a mobile phone. If you alter these parameters, it wouldn't be wise to suggest to your users that your import/export encrypted keys are BIP38 compatible. If you do, you may want to alert them of your parameter changes.
```

Comment:
> @jprichardson if the parameters are changed, won't that stop people being able to decrypt unless they know those parameters?

Of course. But that doesn't mean that we shouldn't prevent the option this case. I should state that I'm a person who usually hates options too. But the standard doesn't define the params. 

The standard should have had the params encoded since they're not fixed.

> I guess I want to ask, is anyone doing this?

I don't know.

> I'm not against making them a static variable, can be changed, but, not part of the "public API" as such.

Why the aversion to just adding an optional param?